packages/cozy-stack-client/src/CozyStackClient.js

```diff
@@ -40,6 +40,12 @@ const normalizeUri = uriArg => {
   return uri
 }
 
+const getResponseData = async response => {
+  const contentType = response.headers.get('content-type')
+  const isJson = contentType && contentType.indexOf('json') >= 0
```

Comment:
I know you copy/pasted this code, but as you are here, I would do an exact `application/json` match here, to prevent any false positive, like "application/kindofjsonbutnotreally"

Reply:
We should probably match `application/vnd.api+json` as well though. What do you think?

Reply:
Absolutely!